ibc-core/ics03-connection/types/src/version.rs

```diff
@@ -117,15 +117,18 @@ impl Display for Version {
 /// counterparty. The returned version contains a feature set with the
 /// intersection of the features supported by the source and counterparty
 /// chains. If the feature set intersection is nil, the search for a
-/// compatible version continues. This function is called in the `conn_open_try`
-/// handshake procedure.
+/// compatible version continues. If no feature set intersection is found
+/// after searching through every supported version, then `None` is returned.
 ///
-/// NOTE: Empty feature set is not currently allowed for a chosen version.
+/// This function is called in the `conn_open_try` handshake procedure.
```

Comment:
Should this comment be reverted? As I see the return's signature remained intact.

Reply:
[29fbe1c](https://github.com/cosmos/ibc-rs/pull/1331/commits/29fbe1cbe4f99bc656fe8b75b509b217d3a358e1)